### PR TITLE
Add `description` field to Issue

### LINKED ISSUE #144: Add short description to issues

### PR DESCRIPTION
**Pull request by kevincianfarini**
Closes #144

### CODE REVIEW

File: backend/src/test/kotlin/org/climatechangemakers/act/feature/util/SqlDriverExtensions.kt [hunk 1]
```diff
@@ -6,12 +6,19 @@ import org.climatechangemakers.act.feature.findlegislator.model.LegislatorPoliti
...
 import org.climatechangemakers.act.feature.findlegislator.model.MemberOfCongress
 
-fun SqlDriver.insertIssue(id: Long, title: String, precomposedTweet: String, imageUrl: String) {
-  execute(0, "INSERT INTO issue(id, title, precomposed_tweet_template, image_url) VALUES(?,?,?,?)", 2) {
+fun SqlDriver.insertIssue(
+  id: Long,
+  title: String,
+  precomposedTweet: String,
+  imageUrl: String,
+  description: String = "description",
```
Comment (EvanHahn) [inline backend/src/test/kotlin/org/climatechangemakers/act/feature/util/SqlDriverExtensions.kt:14]: Should this have a default?

File: backend/src/main/kotlin/org/climatechangemakers/act/feature/issue/manager/DatabaseIssueManager.kt [hunk 2]
```diff
@@ -27,12 +27,18 @@ class DatabaseIssueManager @Inject constructor(
...
+      imageUrl = issue.image_url,
+      description = issue.description,
+      talkingPoints = getIssueTalkingPoints(issue.id)
+    )
   }
 
   override suspend fun getUnfocusedIssues(): List<Issue> = withContext(ioDispatcher) {
     issueQueries.selectUnfocused().executeAsList().map { issue ->
-      async { Issue(issue.id, issue.title, issue.image_url, getIssueTalkingPoints(issue.id)) }
+      async { Issue(issue.id, issue.title, issue.image_url, issue.description, getIssueTalkingPoints(issue.id)) }
```
Comment (EvanHahn) [inline backend/src/main/kotlin/org/climatechangemakers/act/feature/issue/manager/DatabaseIssueManager.kt:41]: Minor: would this be better with keyword arguments like the above?

**Review by EvanHahn — APPROVED** [review]
LGTM. Left a few comments.

File: backend/src/test/kotlin/org/climatechangemakers/act/feature/util/SqlDriverExtensions.kt (hunk 1 above)
Comment (kevincianfarini) [reply backend/src/test/kotlin/org/climatechangemakers/act/feature/util/SqlDriverExtensions.kt:14]: yes, this is testing code. I should probably add defaults to all of the parameters besides `id`, tbh.

**Pull request merged**